expo/gbe/management/commands/sync_audio_downloads.py

```diff
@@ -0,0 +1,139 @@
+from django.core.management.base import BaseCommand
+from django.conf import settings
+import optparse
+import tarfile
+import os
+import shutil
+from gbe.models import (
+    Show,
+    Act,
+)
+from gbe.functions import get_conference_by_slug
+
+
+def make_safe_filename(unsafe_string):
+    '''This is probably wheel reinvention, but I haven't found the wheel
+    If we keep it, it should be improved and turned into a utility function
+    '''
+    return unsafe_string.replace(' ', '_').replace('/', '_')
+
+
+def tarfile_name(show_name, conf_slug):
+    return make_safe_filename("%s_%s.tar.gz" % (conf_slug, show_name))
+
+
+def add_acts_for_show(show_name, conf_slug, tar):
+    '''
+    for acts in show, add track for act to tar
+    '''
+    conference = get_conference_by_slug(conf_slug)
+    acts = Show.objects.get(conference__conference_slug=conf_slug,
+                            title=show_name).get_acts()
+    workdir = make_safe_filename("%s_%s" % (conf_slug, show_name))
+    os.mkdir(workdir)
+
+
+    for act in acts:
+        if act.audio and act.audio.track:
+            fname = os.path.basename(act.audio.track.path)
+            workname = os.path.join(workdir, fname)
+            shutil.copyfile(fname, workname)
```

Comment:
Of note, we should check for the filename before attempting this.  See #675.

Not a blocker